hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/time/TimeFormatter.java

```diff
@@ -0,0 +1,54 @@
+package pl.allegro.tech.hermes.management.infrastructure.time;
+
+import org.joda.time.Period;
+import org.joda.time.format.PeriodFormatter;
+import org.joda.time.format.PeriodFormatterBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+
+@Component
+public class TimeFormatter {
```

Comment:
I think that we can use or at least wrap DateTimeFormatter instead of implementing our own TimeFormatter.